api_app/analyzers_manager/migrations/0057_analyzer_config_mmdb_server.py

```diff
@@ -0,0 +1,80 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {'name': 'Mmdb_server', 'python_module': {'module': 'mmdb_server.MmdbServer', 'base_path': 'api_app.analyzers_manager.observable_analyzers'}, 'description': 'mmdb-server is an open source fast API server to lookup IP addresses for their geographic location, AS number.', 'disabled': False, 'soft_time_limit': 60, 'routing_key': 'default', 'health_check_status': True, 'type': 'observable', 'docker_based': False, 'maximum_tlp': 'RED', 'observable_supported': ['ip'], 'supported_filetypes': [], 'run_hash': False, 'run_hash_type': '', 'not_supported_filetypes': [], 'health_check_task': None, 'model': 'analyzers_manager.AnalyzerConfig'}
```

Comment:
please add a link to repo https://github.com/adulau/mmdb-server in the description

Reply:
noted!